### PR TITLE
Bugfix/collect import specify field

### LINKED ISSUE #1192: Collect import hides attribute (Specify for multientry code in a table)
Collect import does not show SPECIFY text attributes in a table, if the relevancy is linked to multi-entry code attribute

this is case in Collect survey:
image

and this error when trying to publish in Arena:
image

and this is view in Arena editor:
image

### PR DESCRIPTION
**Pull request by SteRiccio**
Fixes #1192

### CODE REVIEW

File: core/expressionParser/expression.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ const isType = (type) => R.propEq('type', type)
 // Return true if the nodeDef can be used in expressions and false otherwise
 export const isValidExpressionType = (nodeDef) =>
   !NodeDef.isEntity(nodeDef) &&
-  !NodeDef.isMultiple(nodeDef) &&
+  //!NodeDef.isMultiple(nodeDef) &&
```
Comment (sorja) [inline core/expressionParser/expression.js:56]: Should this commented line be removed ?

File: core/expressionParser/helpers/evaluator.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ const stdlib = {
   // arity 1+ (arity 0 allowed by JS)
   min: [Math.min, 1, -1],
   max: [Math.max, 1, -1],
+  includes: [(items, value) => items && items instanceof Array && items.includes(String(value)), 2, 2],
```
Comment (sorja) [inline core/expressionParser/helpers/evaluator.js:22]: should use `Array.isArray(items)` replacing ` items && items instanceof Array`

**Review by minotogna — APPROVED** [review]

File: core/expressionParser/expression.js (hunk 1 above)
Comment (SteRiccio) [reply core/expressionParser/expression.js:56]: yes sure!

File: core/expressionParser/helpers/evaluator.js (hunk 2 above)
Comment (SteRiccio) [reply core/expressionParser/helpers/evaluator.js:22]: yes, thanks, you are right

**Review by minotogna — APPROVED** [review]

**Pull request merged**